### PR TITLE
mysql_replication: Add aliases to "master" options, add alternatives to "master" state options, add announcement

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Relates to https://github.com/ansible-collections/community.mysql/issues/145

mysql_replication:
1) add aliases to "master_" options
2) add alternatives to "master" choices of  `state` options
3) add deprecation warnings and announcement

Important: this does NOT contain any breaking changes. I ran the not touched tests with the changes - they passed. Then changed, the changed ones also passed (but you can recheck if any doubts, just checkout to the first commit of the PR).

### CODE REVIEW

**Comment by Andersson007** [comment]
CC @Jorge-Rodriguez @bmalynovytch

File: plugins/modules/mysql_replication.py [hunk 1]
```diff
@@ -92,43 +102,51 @@
...
     - Specifies a colon-separated list of one or more ciphers permitted by the replica for the replication connection.
     - For details, refer to
       L(MySQL encrypted replication documentation,https://dev.mysql.com/doc/refman/8.0/en/replication-solutions-encrypted-connections.html).
     type: str
-  master_auto_position:
+    aliases: [master_ssl_cipher]
+  primary_auto_position:
     description:
     - Whether the host uses GTID based replication or not.
+    - Same as the MASTER_AUTO_POSITION mysql variable.
```
Comment (Jorge-Rodriguez) [inline plugins/modules/mysql_replication.py]:
```suggestion
    - Same as the C(MASTER_AUTO_POSITION) mysql variable.
```

File: plugins/modules/mysql_replication.py [hunk 2]
```diff
@@ -140,16 +158,19 @@
     choices: [current_pos, replica_pos, slave_pos, disabled]
     type: str
     version_added: '0.1.0'
-  master_delay:
+    aliases: [master_use_gtid]
+  primary_delay:
     description:
-    - Time lag behind the master's state (in seconds).
+    - Time lag behind the primary's state (in seconds).
+    - Same as the MASTER_DELAY mysql variable.
```
Comment (Jorge-Rodriguez) [inline plugins/modules/mysql_replication.py]:
```suggestion
    - Same as the C(MASTER_DELAY) mysql variable.
```

**Review by Jorge-Rodriguez — CHANGES_REQUESTED** [review]

**Comment by Andersson007** [comment]
@Jorge-Rodriguez thanks, committed, ready for review

**Review by Jorge-Rodriguez — APPROVED** [review]

**Review by bmalynovytch — APPROVED** [review]

**Comment by Andersson007** [comment]
@Jorge-Rodriguez @bmalynovytch thanks for reviewing!

**Pull request merged**